cmd/lncli/commands.go

```diff
@@ -2974,6 +2974,29 @@ func describeGraph(ctx *cli.Context) error {
 	return nil
 }
 
+var getNodeMetricsCommand = cli.Command{
+	Name:        "getnodemetrics",
+	Category:    "Peers",
```

Comment:
this doesn't exactly pertain to peers, though i see `DescribeGraph` and `GetNodeInfo` already use this category. Perhaps we should group `DescribeGraph`, `GetNodeInfo`, `GetChanInfo`, and `GetNodeMetrics` under `"Graph"`?

Reply:
agree, added a new mini commit for DescribeGraph, GetNodeInfo and GetChanInfo.